src/decode.c

```diff
@@ -916,7 +918,27 @@ decode_R13_R2000 (Bit_Chain *restrict dat, Dwg_Data *restrict dwg)
    */
 
   /*-------------------------------------------------------------------------
-   * Second header, section 3. R13c3-R2000 only.
+   * AcDb:ObjFreeSpace, section 3
+   */
+
+  if (dwg->header.sections > 3
+      && dwg->header.section[SECTION_OBJFREESPACE_R13].address != 0)
```

Comment:
address is usually 0. rather search for the sentinel, and check if object_end equals dat->byte - 53. if there's room for this section before 2ndheader